src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,47 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_OF_ELEMENTS = 10;
+    private final Object[] keys;
+    private final Object[] values;
+    private int size;
+
+    public StorageImpl() {
+        size = 0;
+        keys = new Object[MAX_OF_ELEMENTS];
+        values = new Object[MAX_OF_ELEMENTS];
+    }
+
     @Override
     public void put(K key, V value) {
+        int index = getKeyIndex(key);
+        if (index == -1) {
+            keys[size] = key;
+            values[size++] = value;
+
+        } else {
```

Comment:
Redundant empty line
```suggestion
            values[size++] = value;
        } else {
```